openapi.json

```diff
@@ -268,6 +268,61 @@
           }
         }
       }
+    },
+    "/organizations/{orgIds}": {
+      "delete": {
+        "summary": "Deletes organization data from database.",
+        "description": "All database entries related to the specified organization IDs will be deleted.",
```

Comment:
Please add a warning that this method is available only in debug or testing mode.